security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -644,6 +645,24 @@
                 <dns_nsupdate_key type="TextField">
                     <Required>N</Required>
                 </dns_nsupdate_key>
+                <dns_opnsense_host type="TextField">
+                    <Required>N</Required>
+                    <default>localhost</default>
+                </dns_opnsense_host>
+                <dns_opnsense_port type="TextField">
+                    <Required>N</Required>
+                    <default>443</default>
+                </dns_opnsense_port>
+                <dns_opnsense_key type="TextField">
+                    <Required>Y</Required>
```

Comment:
Actually, none of these should have `<Required>Y</Required>`, because it would break the input validation for all other validation methods.